micro-rdk/src/common/webrtc/api.rs

```diff
@@ -211,6 +212,28 @@ impl WebRtcSignalingChannel {
             }
         }
     }
+
+    pub(crate) async fn send_sdp_error(&mut self, sdp: &WebRtcSdp) -> Result<(), WebRtcError> {
```

Comment:
the function should either be renamed something like "send_error_too_manyc_connection" or update the signature to accept a grpc Status

Reply:
renamed